### PR TITLE
Add Salus security scan to CI.

### PR DESCRIPTION
**Pull request by gksander**
Adds Salus security scan to GH Actions flow.

### CODE REVIEW

File: .github/workflows/static-analysis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: CBPay-JS tests
 on:
   pull_request:
     branches:
-      - 'master'
+      - 'main'
```
Comment (gksander) [inline .github/workflows/static-analysis.yml:6]: Residual from the `master -> main` rename.

Comment (ljharb) [reply .github/workflows/static-analysis.yml:6]: we could also just remove this and let it run on all PRs, not just ones to main

File: .github/workflows/salus-scan.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+jobs:
+  salus_scan_job:
+    runs-on: ubuntu-latest
+    name: Salus Security Scan
+    steps:
+    - uses: actions/checkout@v2
+    - name: Salus Scan
+      id: salus_scan
+      uses: federacy/scan-action@0.1.4
```
Comment (ljharb) [inline .github/workflows/salus-scan.yml:14]: trailing  .

File: .github/workflows/salus-scan.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+on:
+  pull_request:
+    branches:
+      - 'main'
```
Comment (ljharb) [inline .github/workflows/salus-scan.yml:4]: same here

**Review by ljharb — APPROVED** [review]

File: .github/workflows/static-analysis.yml (hunk 1 above)
Comment (gksander) [reply .github/workflows/static-analysis.yml]: ✔️

**Review by ljharb — APPROVED** [review]

**Pull request merged**